k8s/base/jibri-hpa.yaml

```diff
@@ -56,7 +56,7 @@ spec:
         describedObject:
           apiVersion: v1
           kind: Namespace
-          name: jibri
+          name: jitsi
```

Comment:
the filename mentions jibri. Is it a fail?

Reply:
No it's not.
The goal of this `DescribedObject` section is to describe to which object this metric corresponds.
And in this case, this is a metric that correspond to the entire namespace in which jibri is deployed.

The namespace has been renamed to `jitsi` (it was `jibri` before)

Reply:
Ok that's what I missed :+1: